src/style.css

```diff
@@ -1,3 +1,81 @@
+html {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, Arial, sans-serif;
```

Comment:
Consider adding a fallback font family in case 'Roboto' isn't available on the user's system. For example, 'font-family: Roboto, Arial, sans-serif;' could be 'font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;'.